pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -34,6 +34,30 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	// The constant is used if sync_nodes_status=False. NodeController will not proactively
+	// sync node status in this case, but will monitor node status updated from kubelet. If
+	// it doesn't receive update for this amount of time, it will start posting node NotReady
+	// condition. The amount of time when NodeController start evicting pods is controlled
+	// via flag 'pod_eviction_timeout'.
+	// Note: be cautious when changing the constant, it must work with nodeStatusUpdateFrequency
+	// in kubelet. There are several constraints:
+	// 1. nodeStatusUpdateFrequency must be N times less than nodeMonitorGracePeriod, where
```

Comment:
Actually, none of the things where you marked "Fixed" seem to be fixed. Did you forget to upload the new commit?